### DISCUSSION TITLE
Is it possible to run different tasks in a dag at different schedules in airflow?

### DISCUSSION THREAD

**Discussion by gh-jshaik** [apache/airflow#25133] [Q&A]
Is it possible to run different tasks in a dag at different schedules in airflow?

**Comment by potiuk** [comment]
NO

**Reply by gh-jshaik** [reply]
Is there a way or method we can make that possible?

**Reply by potiuk** [reply]
no

**Reply by pingzh** [reply]
@gh-jshaik it is not possible now. maybe you can describe your use cases so that we can better help you.

**Reply by gh-jshaik** [reply]
@pingzh  I have a dag which generates reports for different clients. Here different clients have different requirements. Some need Daily reports and some other clients need monthly reports and some need retrospective reports.

**Reply by potiuk** [reply]
Use https://airflow.apache.org/docs/apache-airflow/stable/howto/dynamic-dag-generation.html?highlight=dynamic @gh-jshaik

**Reply by gh-jshaik** [reply]
@potiuk You suggesting to create different dags for different schedules?

**Reply by potiuk** [reply]
yes
